ncbi/src/main/java/org/reactome/release/Main.java

```diff
@@ -0,0 +1,104 @@
+package org.reactome.release;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.neo4j.driver.v1.*;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+/**
+ * Generates post-release export files for NCBI, UCSC and Europe PMC.
+ * @author jweiser
+ */
+public class Main {
+	private static final Logger logger = LogManager.getLogger();
+
+	/**
+	 * Queries the Reactome Neo4J Graph Database for the current release version and
+	 * generates export files.  The files generated are:
+	 * NCBI Gene XML
+	 *     "Link" XML nodes describing NCBI Gene Identifier relationships to either UniProt entries or Top Level
+	 *     Pathways in Reactome
+	 * NCBI Gene Protein File (not uploaded to NCBI)
+	 *     Tab delimited file of UniProt accession to NCBI Gene identifiers in Reactome
+	 * NCBI Protein File
+	 *     Entries of all UniProt accessions in Reactome associated with an NCBI Gene Identifier (file only contains
+	 *     UniProt accessions)
+	 * UCSC Entity File
+	 *     Entires of all human, rat, and mouse UniProt accessions in Reactome
+	 * UCSC Entity File
+	 *     UniProt accessions (human, rat, and mouse) mapped to the events in which they participate in Reactome
+	 * Europe PMC Profile File
+	 *     Short XML file identifying Reactome as a data provider to Europe PMC
+	 * Europe PMC Link File
+	 *     "Link" XML nodes describing Reactome Pathways connected to PubMed literature references
+	 * @param args Command line arguments for the post-release data files export
+	 * @throws IOException Thrown if unable to create output directory or write files
+	 */
+	public static void main( String[] args ) throws IOException {
+		logger.info("Beginning NCBI, UCSC, and Europe PMC export step...");
+
+		String pathToResources =
+			args.length > 0 ?
+			args[0] :
+			Objects.requireNonNull(
+				Main.class.getClassLoader().getResource("sample_config.properties")
+			).getPath();
+
+		Properties props = new Properties();
+		try {
+			props.load(new FileInputStream(pathToResources));
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+		int version = Integer.parseInt(props.getProperty("reactomeVersion", "67"));
```

Comment:
Why the hard-coded 67?

Reply:
This was a holdover from testing.  It's been removed!